e2e/default_test.go

```diff
@@ -103,22 +146,21 @@ func newSpinAppCR(name, image string) *spinapps_v1alpha1.SpinApp {
 			Executor: "containerd-shim-spin",
 		},
 	}
-	return testSpinApp
 }
 
-func newContainerdShimExecutor(namespace string) *spinapps_v1alpha1.SpinAppExecutor {
```

Comment:
Plz revert (helpers should at least take the namespace for easier conversion to per-test namespaces later)

Reply:
Got it. I was making it consistent with the other helper

Reply:
Oh cool (we should change the other one when we next touch it 😅 )